README.md

```diff
@@ -5,8 +5,11 @@ Generische Gebäudesimulation auf Basis von EnergyPlus™.
 
 GenSim erzeugt anhand einfacher Eingabeparameter ein vollständiges EnergyPlus™-Gebäudemodell, simuliert dieses und gibt anschließend Ergebnisse in Form von Lastprofilen und Jahreswerten zurück.
 
```

Comment:
I think these changes need to be cleanup up, it looks like merge comments are still included.

Reply:
Adressed in a later commit in branch refactor-fix-import-osw